routes/account.js

```diff
@@ -165,4 +165,23 @@ module.exports = function(app) {
     app.get('/account/password_reset', function(req, res, next) {
         res.render('account/reset_password');
     });
+
+    app.delete('/account', function(req, res, next) {
+        var username = req.body.username;
+
+        if (req.user && req.user.isAdmin) {
```

Comment:
this might not work with our new authentication scheme. @balderfer do i need to do something like
```
var auth_token = req.get('Authorization');
jwt.verify(auth_token, app.get('superSecret'), function(err, userId) {
```
and a lookup on userId here?